modules/articles/client/config/articles-admin.client.routes.js

```diff
@@ -0,0 +1,37 @@
+﻿'use strict';
+
+// Setting up route
+angular.module('articles.admin.routes').config(['$stateProvider',
+  function ($stateProvider) {
+    $stateProvider
+    .state('admin.articles', {
+      abstract: true,
+      url: '/articles',
+      template: '<ui-view/>'
+    })
+    .state('admin.articles.list', {
+      url: '',
```

Comment:
this route is inherited from it's parent; `admin.articles` so it doesn't need a url. Essentially, it makes the *list* route the default: `/admin/articles`